tip-wlan/charts/opensync-mqtt-broker/templates/statefulset.yaml

```diff
@@ -46,6 +46,7 @@ spec:
       initContainers:
         - name: {{ include "common.name" . }}-init-dir-ownership-change
           image: alpine:3.6
```

Comment:
While we are at it, it may be worthwhile to remove the hardcoded image and version and define these in values.yaml.